Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -27,6 +27,30 @@
 
 namespace itk
 {
+
+
+/** \class Analyze75Flavor
+ * \ingroup ITKIONIFTI
+ * Enum used to define way to treat legacy Analyze75 files
+ */
+enum class Analyze75Flavor : uint8_t
```

Comment:
I guess wrapping is needed for this class
`itkNiftiImageIO: warning(4): ITK type not wrapped, or currently not known: itk::Analyze75Flavor`
A recent example of how to do it:
https://github.com/InsightSoftwareConsortium/ITKMontage/commit/c65afeef7a5f35e0975d2fc07908552a8b6ca908#diff-517bbaeb1b2d54fcf3da514456be02e7R5